src/controller/invoice.controller.js

```diff
@@ -0,0 +1,124 @@
+import invoiceModel from "../model/invoice.model.js";
+import { validationResult } from "express-validator";
+import { response } from "express";
+
+// Obtener todas las facturas
+export const getInvoices = async (req, res = response) => {
+  try {
+    const invoices = await invoiceModel.find();
+    res.status(200).json({
+      invoices,
+    });
+  } catch (error) {
+    res.status(500).json({
+      error: "Error al obtener las facturas",
+    });
+  }
+};
```

Comment:
esta bien, pero necesito que te rijas por este estlio y esta convencion, ademas lo necesito TODO en ingles. las variables, los nombres de los meodos, y los errores
```suggestion
export const getInvoices = async (req, res = response) => {
  const { limit, page } = req.query;
  const query = { tp_status: "ACTIVE" };
  const [total, invoices] = await Promise.all([
    InvoiceModel.countDocuments(query),
    InvoiceModel.find(query).skip(Number(page) * Number(limit)).limit(Number(limit))
  ]);
  res.status(200).json({
    total,
    invoices,
  });
};
```

de momento no estamos usando try catch en los controladores. esto para obligar aquellos a hacer buenos middlewares, lo mismo con vos, no vamos a trabajar con try catch dentro de los controladores, en cambio, vamos a centrarnos mucho en hacer buenos middlewares.